NEWS

```diff
@@ -11,6 +11,333 @@
 ### Features:
 ### Bugfixes:
 
+## 1.14.0 (January 12, 2023)
```

Comment:
should be RC and not release ? @yosefe

Reply:
@brminich we start with RC and remove RC from the release name just before official release.